charts/ocis/templates/_common/_tplvalues.tpl

```diff
@@ -42,6 +42,7 @@ Adds the app names to the scope and set the name of the app based on the input p
   {{- $_ := set .scope "appNameEventhistory" "eventhistory" -}}
   {{- $_ := set .scope "appNameFrontend" "frontend" -}}
   {{- $_ := set .scope "appNameGateway" "gateway" -}}
+  {{- $_ := set .scope "appNameWebfinger" "webfinger" -}}
```

Comment:
this list looks like the services are sorted alphabetically. Probably it's a good idea to add webfinger after web.